### ISSUE TITLE
Spectral Relay Overlap Has No "Cannibalism" Penalty

### ISSUE THREAD

**Issue by VT-14** [HellFirePvP/AstralSorcery#1405]
Tested on Minecraft 1.12.2 and 1.15.2.

In previous versions, placing Starlight Collectors, including Spectral Relays, close together would cause them to significantly interfere with each other and produce far less Starlight than you would get from spreading them apart.

On the versions I tested, having Spectral Relays so close they are sharing blocks in their structures produced just as much Starlight in a Celestial Altar as the same number of Relays do when spread far apart.

Asking on the Discord implied this wasn't an intentional change.  I have not tested any other Starlight Collectors.

****

**1.12.2**

Album: https://imgur.com/a/LbTQ7wM

Full Mod List:
* Forge: 14.23.5.2854
* Astral Sorcery: 1.12.2-1.10.27
* Baubles: 1.12-1.5.2
* Just Enough Items: 1.12.2-4.16.1.302

Log: https://gist.github.com/VT-14/37a3a2c6b178fd71aa4a1598358cb565
No debug crash done.

****

**1.15.2**

Album: https://imgur.com/a/RN1NzCV

Full Mod List:
* Forge: 31.2.44
* Astral Sorcery: 1.15.2-1.12.9
* ObserverLib: 1.15.2-1.2.0.v60
* Curios API: FORGE-1.15.2-2.0.2.6
* Just Enough Items: 1.15.2-6.0.3.15

Log: https://gist.github.com/VT-14/1ab1d40bb1f32422c82a1a9a392aa233
Debug Crash: https://gist.github.com/VT-14/b34ad275927a6b9fb02a73283c70decc

**Comment by HellFirePvP** [comment]
Right. I forgot that for 1.15 :P
In case this isn't a thing in 1.12, i won't bother with it anymore personally.

**Comment by HellFirePvP** [comment]
Also, something is off with the tileentitytype registration there...
https://gist.github.com/VT-14/1ab1d40bb1f32422c82a1a9a392aa233#file-1-15-2-as-spectral-relays-logs-latest-log-L1719-L1736

**Comment by VT-14** [comment]
Yes, I loaded up the same world with the hopes that the some or all of the AS structures would be saved.  They were not, though I was not too disappointed from what I know of the 1.13 'flattening' update.  Plus, I checked with the Fosic Resonator (I intentionally set this up in an area without the high starlight cloud particles), and the spot I had been using had become a high intensity spot, so I had to move anyway.

I could recreate this on a fresh world, but your edit makes it look like that isn't necessary.
